pkg/build/api/types.go

```diff
@@ -911,6 +916,9 @@ type GenericWebHookEvent struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
+
+	// StrategyOptions contains additional strategy-specific options for the build
+	StrategyOptions *BuildStrategyOptions
```

Comment:
the generic struct for the BuildRequest object is fine, but i don't see why you're using the struct here, within the dockerstrategy object.  (in fact you're not using it, but you've still defined the field)